teamManager/teamManager.py

```diff
@@ -1122,6 +1122,7 @@ async def _find_team_name(self, ctx, franchise_role, tier_role):
         for team in teams:
             if await self._roles_for_team(ctx, team) == (franchise_role, tier_role):
                 return team
+        return
```

Comment:
why explicit `return` -- linting?

Reply:
I'm not sure. That was in previously, and wasn't my LOC. I suppose it might be a good idea to determine some coding standards one way or another. I'm personally partial to an explicit `return None` for reasons I shared in other comments, but I don't think we need to go back and fix all of them right now.